Cargo.toml

```diff
@@ -268,7 +268,6 @@ indexmap = "1.0.0"
 itertools = "0.10.3"
 ndarray = "0.15.0"
 nom = "7.0.0"
-once_cell = "1.15.0"
```

Comment:
question: was there no code using this?

Reply:
There was, and it's updated in this PR. Ignore all the baked data.

Reply:
ah it's just not showing up on my phone because of the baked data. I'll wait until I have a computer